### ISSUE TITLE
[BUG] Should ignore .dap-breakpoints and .extension

### ISSUE THREAD

**Issue by ncihnegn** [hlissner/doom-emacs#2273]
**What did you expect to happen?**
Clean git status after installing dap

**What actually happened?**
```
➜  .emacs.d git:(db9bc763e) ✗ git status -sb
## HEAD (no branch)
?? .dap-breakpoints
?? .extension/
```

**Comment by hlissner** [comment]
Please include your `M-x doom/info` so I can determine if you have the `:tools debugger` module enabled, or if you are installing lsp manually.

**Comment by ncihnegn** [comment]
Yes, both debugger and lsp.
<details>

```
emacs   version    26.3
        features   NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS THREADS
        build      Dec 03, 2019
        buildopts  (--enable-locallisppath=/usr/local/share/emacs/site-lisp --infodir=/usr/local/Cellar/emacs-mac/emacs-26.3-z-mac-7.8/share/info/emacs --prefix=/usr/local/Cellar/emacs-mac/emacs-26.3-z-mac-7.8 --with-mac --enable-mac-app=/usr/local/Cellar/emacs-mac/emacs-26.3-z-mac-7.8 --with-gnutls)
        windowsys  mac
        daemonp    server-running
doom    version    2.0.9
        build      HEAD, origin/develop, origin/HEAD db9bc763e 2019-12-29 19:24:55 -0500
        dir        ~/.doom.d/
system  type       darwin
        config     x86_64-apple-darwin19.0.0
        shell      /bin/zsh
        uname      Darwin 19.2.0 Darwin Kernel Version 19.2.0: Sat Nov  9 03:47:04 PST 2019; root:xnu-6153.61.1~20/RELEASE_X86_64 x86_64
config  envfile    nil
        elc-files  0
        modules    (:completion company ivy :ui doom doom-dashboard hl-todo modeline nav-flash ophints (popup +all +defaults) vc-gutter vi-tilde-fringe window-select workspaces :editor (evil +everywhere) file-templates fold multiple-cursors rotate-text snippets :emacs dired electric ibuffer vc :tools debugger (eval +overlay) flycheck (lookup +docsets) lsp macos magit :lang agda assembly (cc +lsp) clojure coq csharp data elixir elm emacs-lisp erlang ess fsharp (haskell +lsp) idris julia kotlin latex lua markdown nim ocaml (org +dragndrop +ipython +pandoc +present) purescript python racket ruby rust scala scheme sh swift web :config (default +bindings +smartparens))
        packages   (n/a)
        elpa       (n/a)
```
</details>

**Comment by hlissner** [comment]
Thanks for the prompt response!

It seems the `dap--breakpoints-file` was renamed to `dap-breakpoints-file` at some point. I've corrected this in eba070d. However, `dap-utils-extension-path` is already set to use `~/.emacs.d/.local/etc/dap-extension/`, so I'm not sure why it's creating that directory for you.

Can you delete the directory and confirm it is still being created?

**Comment by ncihnegn** [comment]
Can't verify before f8ff50565 is fixed.
